Readme_files/Programming.md

```diff
@@ -1,99 +1,109 @@
-<img src="/Readme_files/stlink_connection.jpg?raw=true"><br><br>
+<img src="/Readme_files/stlink_connection.jpg?raw=true">
 
 ## Clone detection
 
-A lot of sellers started using relabeled STM32 Clones, making detection hard.<br>
-Thankfully, most can be easily detected by checking their CPU ID (DBGMCU_ID) code.<br><br>
-Connect the ST-Link, write 0xE0042000 in Address, click Connect to target.<br>
+A lot of sellers started using relabeled STM32 clones, making detection hard.<br>
+Thankfully, most can be easily detected by checking their CPU ID (DBGMCU_ID) code by folowing the procedure:
+
+- Connect the ST-Link,
+- Write `0xE0042000` in "Address" field,
+- Write `0x8` in "Size" field,
+- Click "Connect to target" button.
+
 A genuine STM32F101/102/103Cx/Rx should have any of these codes:
-    00006410 20006410 20016410 20036410
-    
-<img src="/Readme_files/dbgmcu_id.png?raw=true"><br><br>
+
+> 00006410<br>
+> 20006410<br>
+> 20016410<br>
+> 20036410
+
+<img src="/Readme_files/dbgmcu_id.png?raw=true">
+
 
 ## Programming with STM32 ST-LINK Utility
 
-Download the binary **STM32SolderingStation.bin** for your station from [Releases](https://github.com/deividAlfa/stm32_soldering_iron_controller/releases), and flash it using ST-Link.<br><br>
+Download the binary **STM32SolderingStation.bin** for your station from [Releases](https://github.com/deividAlfa/stm32_soldering_iron_controller/releases), and flash it using ST-Link.
 
 ### Coming from original firmware
 
 Usually the STM32 comes read-protected. Follow this workflow to program it.<br>
 (Click for bigger picture)<br>
-<img src="/Readme_files/st-link_programming.png?raw=true"><br><br>
+<img src="/Readme_files/st-link_programming.png?raw=true">
 
 ### Upgrading
 
 Just open and program the new binary.<br>
 Because this firmware stores the settings in the flash, don't make a full chip erase!<br>
 <img src="/Readme_files/upgrade.png?raw=true"><br>
 
-In any case, the firmware will check the settings and reset them if not valid.<br><br>
+In any case, the firmware will check the settings and reset them if not valid.
 
 ### STM32 not recognized error
-Some STM32 have the debug port disabled, in this case the only way to access is connecting nRST pin to the ST-Link.<br>
-There're several ST-Link clones with a flaw in nRST pin, which is wired wrong by design, and won't able to connect.<br> 
-Temporaly short nRST (STM32 pin 7) to gnd, click "connect" button, wait 1-2 seconds and release nRST, now it should recognice it.<br>
-Usually nRST is connected to a capacitor, only requiring to short the capacitor as showed in this picture:<br>
-<img src="/Readme_files/stlink_force_rst.jpg?raw=true"><br><br>
+Some STM32 have their debug port disabled, in this case the only way to access is connecting `nRST` pin to the ST-Link.<br>
+There're several STM32 clones with a flaw in `nRST` pin, which is wired wrong by design, and won't able to connect.<br>
```

Comment:
The original text is "ST-Link clones" I suppose the intend was to be "STM32 clones". Am I right?

Reply:
Nope, it really means ST-Link clones from AliExpress and such, some had wrong pcb layout, the pin that toggled nRST wasn't connected to the programming connector, so it didn't work.

Reply:
Here you can see what I mean:
https://ziutek.github.io/2018/03/26/stlink.html

Reply:
I fixed that.